### PR TITLE
Add icon for AusweisApp2

### PR DESCRIPTION
**Pull request by Sir-Photch**
## Description

Adding icon for AusweisApp2. The "2" is not a typo!

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`com.governikus.ausweisapp2`)

### Icons linked
* App Name (linked `com.governikus.ausweisapp2` to `@drawable/ausweisapp2`)
* App Name (linked `com.governikus.ausweisapp2.dev` to `@drawable/ausweisapp2`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -52,6 +52,7 @@
     <icon drawable="@drawable/audiofx" package="org.lineageos.audiofx" name="AudioFX" />
     <icon drawable="@drawable/aurora_droid" package="com.aurora.adroid" name="Aurora Droid" />
     <icon drawable="@drawable/aurora_store" package="com.aurora.store" name="Aurora Store" />
+    <icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2" name="AusweisApp2" />
```
Comment (daywalk3r666) [inline app/src/main/res/xml/grayscale_icon_map.xml:55]: <icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2.dev" name="AusweisApp2" />

This is the beta version. Maybe you can add it

Comment (Sir-Photch) [reply app/src/main/res/xml/grayscale_icon_map.xml:55]: Wdym exactly? `com.governikus.ausweisapp2` is a beta version?

Comment (Chefski) [reply app/src/main/res/xml/grayscale_icon_map.xml:55]: com.governikus.ausweisapp2.dev is the beta package name if you wanna add it too

Comment (Sir-Photch) [reply app/src/main/res/xml/grayscale_icon_map.xml:55]: sure give me 1 second

Comment (daywalk3r666) [reply app/src/main/res/xml/grayscale_icon_map.xml:55]: Sry forgot `` 
`<icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2.dev" name="AusweisApp2" />`

**Review by Chefski — CHANGES_REQUESTED** [review]
Opacity is not at 100%. Mind fixing that?

**Review by Chefski — APPROVED** [review]

**Pull request merged**